<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boardGenerator</class>
 <widget class="QMainWindow" name="boardGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>boardGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="display_image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="xCount">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Markerzahl X-Achse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Markerzahl Y-Achse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yCount">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kantenlänge eines Markers (in px)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="markerLength">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="markerSeparation">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abstand zwischen Markern (in px)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dictionary">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bibliothek Index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StartIndex">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateBoard">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Board generieren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBoard">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Board speichern</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>401</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Die Darstellung ist auf 500x500px skaliert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tatsächliche Größe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="actualSize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
